main.js

```diff
@@ -39,6 +38,14 @@ app.get('/follow', (req, res) => {
   twitter.postFollow({}, error, success);
 })
 
+app.get('/ids', (req, res) => {
+  database.getUserIds('clients')
+    .then((result) => {
+      console.log(result[1].id);
+      res.send(result[1].id);
```

Comment:
The result array contains 1,500 ids so this was a way to test functionality without dumping a lot of text onto the screen.